src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/PipeListener.cs

```diff
@@ -19,13 +19,15 @@ public PipeListener(ServiceContext serviceContext) : base(serviceContext)
         /// <summary>
         /// Creates the socket used to listen for incoming connections
         /// </summary>
-        protected override UvStreamHandle CreateListenSocket()
+        protected override void CreateListenSocket()
```

Comment:
Yeah. I think I rejected it too hastily because I wanted everything to be disposed as normal in the `DisposeAsync` path. Not having `ListenSocket` prevents most of `DisposeAsync` from running, but that should be fine. I'll change it.